jwst/extract_1d/extract.py

```diff
@@ -742,6 +742,9 @@ def copy_keyword_info(slit, slitname, spec):
         "quadrant",
         "slit_xscale",
         "slit_yscale",
+        "wavelength_corrected",
+        "pathloss_correction_type",
```

Comment:
Should the pathloss_correction_type also be copied to the IFU x1d?  Currently it looks like it's not since the IFU runs through its own extraction.

Reply:
Ah good point, I had only looked at the IFU cal files, but didn't check the cubes (which I assume this keyword is not propagated to, so it couldn't be easily passed to the x1d).  And yes, it's likely not that critical for IFU, regardless, (especially since the logs are now stored).

Happy to move forward with the PR as is.